tests/fixtures/Cargo_v2.lock

```diff
@@ -0,0 +1,220 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
Do we know which rust versions correspond to the different `Cargo.lock` file versions?

I'd like to get an idea of how important it is to support the old versions.

Reply:
I don't see which cargo versions associate with which lockfile version, but the change from version 2 to version 3 was February of 2021.

Reply:
So that's rust 1.50 ... Probably old enough that we don't really have to worry about it.

No change required here, just wanted to know. Thanks